tests/conftest.py

```diff
@@ -51,9 +51,8 @@ def __init__(self, name, parent, url, ref, verify):
         self.verify = verify
 
     def runtest(self):
-        subprocess.check_call([
-            'jupyter-repo2docker',
-            '--ref', self.ref,
-            self.url,
-            '--',
-        ] + shlex.split(self.verify))
+        if self.ref is not None:
+            cmd = ['jupyter-repo2docker', '--ref', self.ref, self.url, '--']
+        else:
+            cmd = ['jupyter-repo2docker', self.url, '--']
```

Comment:
Needed to support the new test case that has no ref.

Reply:
This is also why tests passed, since --ref was before self.url!